### ISSUE TITLE
Extreme performance loss translating signs (mostly modded ones) 1.16.5

### ISSUE THREAD

**Issue by NighthowlerStudios** [ringosham/TranslateMod#19]
https://spark.lucko.me/NtrZXsXOss

the above is a possible incompat with Performant mod, but I doubt it

also causes logspam with optifine render regions enabled:

[13:59:20] [Render thread/ERROR]: [OptiFine] OpenGL error: 1281 (Invalid value), at: Copy VBO range
[13:59:20] [Render thread/ERROR]: OpenGL API ERROR: 1281 (GL_INVALID_VALUE error generated. neither <first> nor <count> can be negative.)

**Comment by NighthowlerStudios** [comment]
![image](https://user-images.githubusercontent.com/82226641/139439964-8c4e4972-a533-49fa-b10d-ef914a1a03dd.png)
focused on your mod

**Comment by ringosham** [comment]
This seems to be an easy fix.

**Comment by NighthowlerStudios** [comment]
sounds to me like you don't cache results?

**Comment by ringosham** [comment]
Nah. This is just a small mistake. It should not process things on the server at all in the first place. Everything should stay client side.
I'm also in the processing of porting it to 1.17.1 so updates might take a while to release

**Comment by NighthowlerStudios** [comment]
nothing has been done to the 1.16.5 to fix the issue, is this issue abandoned or patched or only in newer mc versions?
